src/biokbase/narrative/jobs/job.py

```diff
@@ -359,9 +384,9 @@ def state(self, force_refresh=False, exclude=JOB_INIT_EXCLUDED_JOB_STATE_FIELDS)
             state = self.query_ee2_state(self.job_id, init=False)
             self._update_state(state)
 
-        return self._internal_state(exclude)
+        return self.static_state(exclude)
 
-    def _internal_state(self, exclude=None):
+    def static_state(self, exclude=None):
```

Comment:
maybe change this to `current_state` or `cached_state` to better reflect what is being returned?